src/runtime/linux.rs

```diff
@@ -825,6 +827,23 @@ pub enum LinuxNamespaceType {
     Time = 0x00000080,
 }
 
+impl Display for LinuxNamespaceType {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let print = match self {
+            LinuxNamespaceType::Mount => "mnt",
+            LinuxNamespaceType::Cgroup => "cgroup",
+            LinuxNamespaceType::Uts => "uts",
+            LinuxNamespaceType::Ipc => "ipc",
+            LinuxNamespaceType::User => "user",
+            LinuxNamespaceType::Pid => "pid",
+            LinuxNamespaceType::Network => "net",
+            LinuxNamespaceType::Time => "time",
+        };
+
+        write!(f, "{print}")
+    }
+}
```

Comment:
I think we could use `#[strum(to_string = "mnt")]` and such in replacement for that manual implementation, right?

Reply:
Yes, Thx @saschagrunert, I will correct it.

Reply:
@saschagrunert  Fixed, PTAL! Thx